engine/src/flutter/flow/stopwatch_dl.cc

```diff
@@ -50,20 +50,21 @@ void DlStopwatchVisualizer::Visualize(DlCanvas* canvas,
   // Prepare a path for the data; we start at the height of the last point so
   // it looks like we wrap around.
   {
+    auto bar_left = x;
     for (auto i = 0u; i < stopwatch_.GetLapsCount(); i++) {
+      auto time_ms = stopwatch_.GetLap(i).ToMillisecondsF();
       auto const sample_unit_height =
-          (1.0 - UnitHeight(stopwatch_.GetLap(i).ToMillisecondsF(),
-                            max_unit_interval));
+          height * UnitHeight(time_ms, max_unit_interval);
 
-      auto const bar_width = width * sample_unit_width;
-      auto const bar_height = height * sample_unit_height;
-      auto const bar_left = x + width * sample_unit_width * i;
+      auto const bar_top = bottom - sample_unit_height;
```

Comment:
```suggestion
      const auto bar_top = bottom - sample_unit_height;
```

https://google.github.io/styleguide/cppguide.html#Use_of_const

`auto const` is used a lot in this file but it doesn't match the usage of const across the wider codebase.

Also, please remove the auto.  I'm not sure if we are doing floating point math or integer math here.

Reply:
The compiler seems to allow `Scalar foo = some_double [op] some_other_double` - not sure if we should be explicit about a cast there or just rely on the compiler. It passes clang-tidy...

Reply:
Yea that's c++ for yea.  I bet there is a flag to turn off implicit casts but would probably be difficult to turn on at this point.